verible/verilog/analysis/checkers/constraint-name-style-rule.h

```diff
@@ -26,22 +30,36 @@
 namespace verilog {
 namespace analysis {
 
-// ConstraintNameStyleRule check each constraint name follows the correct naming
-// convention.
-// The constraints should be named with lower_snake_case and end with _c.
+// Lower snake case, ends with `_c`
+#define kSuffix "([a-z0-9]+_)+c"
```

Comment:
Instead of a macro, can this be a `static constexpr absl::string_view kSuffix` in the private section of ConstraintNameStyleRule instead ?

Reply:
Indeed, thanks! Before `kPrefix` was here too and that "needed" to be a define to do this stupid trick https://github.com/IEncinas10/verible/blob/de368dbc5d0a46ad14763854ee105f89a65d476a/verible/verilog/analysis/checkers/constraint-name-style-rule_test.cc#L73